### PR TITLE
MM-50750 Remove unused HelpController component

### PR DESCRIPTION
**Pull request by hmhealey**
#### Summary
The route that these components are on hasn't been linked to since the formatting bar was added to the post textbox. People could still get to them by typing the URL directly, but I don't imagine anyone has done that. Removing them deletes a bunch of code and removes a bunch of translation strings to make translators' lives easier

#### Ticket Link
MM-50750

#### Release Note
```release-note
Removed in-app help pages that were no longer accessible
```

### CODE REVIEW

**Comment by hmhealey** [comment]
@cwarnermm I added you as a reviewer just in case anyone cared about these pages from the documentation side

**Review by koox00 — APPROVED** [review]

File: webapp/channels/src/components/root/root.tsx [hunk 1]
```diff
@@ -580,10 +578,6 @@ export default class Root extends React.PureComponent<Props, State> {
                         path={'/claim'}
                         component={ClaimController}
                     />
-                    <HFTRoute
-                        path={'/help'}
-                        component={HelpController}
-                    />
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/root/root.tsx:586]: i love these kinds of PRs

**Review by M-ZubairAhmed — APPROVED** [review]

**Review by larkox — APPROVED** [review]
The changes LGTM, but I wonder if this help is still needed. Sure, it is now not accessible, and maybe people haven't complained, but 0/5 on whether we still need something like this somewhere.

**Comment by hmhealey** [comment]
Good call. I'll add @itao just in case we want to keep these translations around. I would hope people don't need these since we have the formatting bar for people to use now

**Review by cwarnermm — APPROVED** [review]
This is a fine looking PR :)

**Pull request merged**